### PR TITLE
fix(install): resolve #224

### LINKED ISSUE #224: Jinja variable 'config' is undefined
Hi,

I ran a state apply on this formula, docker dependencies got updated:
```
          ID: docker-package-dependencies
    Function: pkg.installed
      Result: True
     Comment: The following packages were installed/updated: python3-pip
              The following packages were already installed: apt-transport-https, python3-apt
```

However it errors out with:
```
----------
          ID: docker-config
    Function: file.managed
        Name: /etc/default/docker
      Result: False
     Comment: Unable to manage file: Jinja variable 'config' is undefined
     Started: 05:20:04.208177
    Duration: 149.465 ms
     Changes:
----------
          ID: docker-service
    Function: service.running
        Name: docker
      Result: False
     Comment: One or more requisite failed: docker.install.docker-config
     Started: 05:20:04.359830
    Duration: 0.022 ms
     Changes:
```

Looking at the source code I cannot see anywhere where `config` is defined.

### PR DESCRIPTION
**Pull request by noelmcloughlin**
Resolves #224 error

### CODE REVIEW

File: docker/install.sls [hunk 1]
```diff
@@ -55,6 +55,8 @@ docker-config:
     - template: jinja
     - mode: 644
     - user: root
+    - context:
+      config: {{ docker.config | json }}
```
Comment (myii) [inline docker/install.sls]: The indentation is significant, as pointed out by @daks earlier (I'll see if I can find the reference):

```suggestion
        config: {{ docker.config | json }}
```

Comment (myii) [reply docker/install.sls]: @noelmcloughlin Here you go: https://github.com/orgs/saltstack-formulas/projects/1#card-20142638.

**Comment by noelmcloughlin** [comment]
Thanks @myii and @daks

**Comment by myii** [comment]
You're welcome, @noelmcloughlin.

**Pull request merged**